src/features/DisplayControls/NursingTasks/utils/NursingTasksUtils.js

```diff
@@ -52,11 +57,30 @@ export const ExtractMedicationNursingTasksData = (
           minute: "2-digit",
           hourCycle: "h23",
         }),
-      });
+      }
+
+      if ( (filterValue.id === "completed" && slot.status === "COMPLETED") || (filterValue.id === "allTasks" && slot.status === "COMPLETED") ) {
+        completedExtractedData.push({
+          ...slotInfo,
+          administeredTimeInEpochSeconds: administeredDateTime,
+          administeredTime: administeredTimeInDate!== '' ? administeredTimeInDate.toLocaleTimeString([], {
+            hour: "2-digit",
+            minute: "2-digit",
+            hourCycle: "h23",
+          }) : '',
+        });
+      } else if ( (filterValue.id === "pending" && slot.status === "SCHEDULED") || (filterValue.id === "allTasks" && slot.status === "SCHEDULED") ) {
+        pendingExtractedData.push({
```

Comment:
We can directly do pendingExtractedData.push(slotInfo) instead of spread operator

Reply:
Sure